Makefile

```diff
@@ -121,6 +123,28 @@ endif
 .PHONY: update-embedded-full-documentation
 update-embedded-full-documentation: doc
 	rsync -a --delete doc/html/ lxd/gendocs/html/
+	# Remove the .doctrees directory, as it is useless for us and will reduce the size.
+	rm -Rf lxd/gendocs/html/.doctrees
+
+	# Minify content
+	@echo "Minifying HTML files..."
+	@find lxd/gendocs/html/ -name '*.html' -type f -print0 | xargs -0 -n 1 -P `nproc` -I {} $(HTMLMINIFY) -o {} {}
```

Comment:
Could the HTML, JS, CSS and SVG minifiers pipe to `gzip -9` to write those assets only once?